apps/cowswap-frontend/src/modules/account/hooks/useToggleAccountModal.ts

```diff
@@ -7,3 +7,9 @@ export function useToggleAccountModal() {
 
   return () => updateAccountModalState((prev) => ({ isOpen: !prev.isOpen }))
 }
+
+export function useCloseAccountModal() {
+  const updateAccountModalState = useSetAtom(updateAccountModalStateAtom)
+
+  return () => updateAccountModalState({ isOpen: false })
```

Comment:
Should it be wrapped into `useCallback`?

Reply:
Good catch!
Based on the same hook above, will update it too.